### PR TITLE
fix: Grid A11y improvements

### PR DESCRIPTION
**Pull request by josebui**
## Description

- Added custom unsorted icon
- Unosrted icon always visible
- Removed filter item from column menu
- Remove column menu if not sortable
- Removed title from header button
- Added list markup for pagination
- Fixed focus for pagination buttons
- Wrap text inside each cell
- Fixde tab navigation for tables, now just one tab will get to the content and the rest of the navigation should happen with arrow keys

### Checklist
- [x] Corresponding issue has been opened
- [ ] New tests added
- [ ] English strings reviewed and copyedited
- [ ] Strings are localized
- [ ] Verified on mobile
- [ ] Verified on desktop
- [x] Verified accessibility ([Accessibility - Web applications](https://docs.google.com/document/d/1VwPDyLqw7r_iISMgDZr1FdAmAqqOIIVIxQqsYA-HaHE/edit?usp=sharing))

### Related Issues
Fixes #....

Airtable: 
- https://airtable.com/appGFQz3URVU5GgXt/tblsA2qhrnKiImmOl/viwwZCSHVM5CZkPHM/recrQuSBQuOuBIK0j?blocks=hide
- https://airtable.com/appGFQz3URVU5GgXt/tblsA2qhrnKiImmOl/viwwZCSHVM5CZkPHM/reczVv78TgCwJ2kvO?blocks=hide
- https://airtable.com/appGFQz3URVU5GgXt/tblsA2qhrnKiImmOl/viwwZCSHVM5CZkPHM/recr498p9q2VaXrJX?blocks=hide
- https://airtable.com/appGFQz3URVU5GgXt/tblsA2qhrnKiImmOl/viwwZCSHVM5CZkPHM/recBCBGjDIFXWwegT?blocks=hide
- https://airtable.com/appGFQz3URVU5GgXt/tblsA2qhrnKiImmOl/viwwZCSHVM5CZkPHM/recME7VMu0oObN9qb?blocks=hide
- https://airtable.com/appGFQz3URVU5GgXt/tblsA2qhrnKiImmOl/viwwZCSHVM5CZkPHM/recqlua6mqKIjElo3?blocks=hide
- https://airtable.com/appGFQz3URVU5GgXt/tblsA2qhrnKiImmOl/viwwZCSHVM5CZkPHM/recxR7OhTXPP4i5qy?blocks=hide

### CODE REVIEW

**Review by paulschreiber — APPROVED** [review]

File: src/localization/locales/en-US.json [hunk 1]
```diff
@@ -646,6 +646,13 @@
         "table_search_filter_results_other": "{{rows.length}} matches found for “{{query}}“",
         "table_search_filter_results_one": "{{rows.length}} match found for “{{query}}“",
         "table_search_filter_clear": "Clear search",
+        "table_pagination_label": "Table pagination",
+        "table_pagination_previous_page": "Go to previous page",
+        "table_pagination_next_page": "Go to next page",
+        "table_pagination_displayed": "{{from}} - {{to}} of {{count}}",
+        "table_sort_desc": "Sort by DESC",
+        "table_sort_asc": "Sort by ASC",
```
Comment (paulschreiber) [inline src/localization/locales/en-US.json:654]:
```suggestion
        "table_sort_desc": "Sort descinding",
        "table_sort_asc": "Sort ascending",
```

Comment (josebui) [reply src/localization/locales/en-US.json]: @paulschreiber Is that a typo for descending?

Comment (paulschreiber) [reply src/localization/locales/en-US.json]: Yes. fixed.

File: src/localization/locales/en-US.json [hunk 2]
```diff
@@ -646,6 +646,13 @@
         "table_search_filter_results_other": "{{rows.length}} matches found for “{{query}}“",
         "table_search_filter_results_one": "{{rows.length}} match found for “{{query}}“",
         "table_search_filter_clear": "Clear search",
+        "table_pagination_label": "Table pagination",
+        "table_pagination_previous_page": "Go to previous page",
+        "table_pagination_next_page": "Go to next page",
+        "table_pagination_displayed": "{{from}} - {{to}} of {{count}}",
+        "table_sort_desc": "Sort descinding",
```
Comment (paulschreiber) [inline src/localization/locales/en-US.json:653]:
```suggestion
        "table_sort_desc": "Sort descending",
```

**Pull request merged**